docs/apis-tools/web-modeler-api/tutorial.md

```diff
@@ -0,0 +1,172 @@
+---
+id: modeler-api-tutorial
+title: Tutorial
+sidebar_position: 3
+slug: /apis-tools/web-modeler-api/tutorial
+description: "Step through an example to view your existing clients, create a client, view a particular client's details, and delete a client."
+---
+
+In this tutorial, we'll step through examples to highlight the capabilities of the Web Modeler API, such as creating and deleting a file.
+
+## Prerequisites
+
+- If you haven't done so already, [create a cluster](/components/react-components/create-cluster.md).
```

Comment:
Do you actually need a cluster? I don't think you do anything with it in this tutorial, do you?

Reply:
Reopening this - I need to see where a cluster would be necessary in this tutorial. Can you explain why you would need a cluster?

Reply:
Adjusted in latest commit 👍